ui/lib/ldap/addon/routes/roles/index.ts

```diff
@@ -58,4 +79,11 @@ export default class LdapRolesRoute extends Route {
       { label: resolvedModel.backendModel.id },
     ];
   }
+
+  resetController(controller: LdapRolesController, isExiting: boolean) {
+    if (isExiting) {
```

Comment:
not sure if this matters, but I set my defaults to null.

Reply:
I think I did that but it still had the keys in the url like `currentPage=` for example. I could be wrong but I know that `undefined` completely removes them from the url.